instruments/models.py

```diff
@@ -269,15 +272,24 @@ def to_topic_dict(self, language="en"):
             concept_key="concept_%s" % concept_name,
         )
 
-    def comparison_string(self, to_string=False):
-        cs = ["Question %s" % self.title()]
+    def comparison_string(self, to_string=False, wrap=50):
+        cs = ["Question: %s" % self.title()]
         for item in self.get_source().get("items", []):
             cs += [
                 "",
-                "Item: %s (scale: %s)" % (item.get("value"), item.get("scale")),
+                "Item: %s (scale: %s)" % (item.get("item"), item.get("scale")),
                 item.get("text", ""),
             ]
             cs += ["%s: %s" % (a["value"], a["label"]) for a in item.get("answers", [])]
+        if wrap:
+            cs_temp = [ textwrap.wrap(line, wrap) for line in cs ]
+            cs = []
+            for wrap in cs_temp:
+                if wrap == []:
+                    cs.append("")
+                else:
+                    cs += wrap
+            #cs = [ line for wrap in cs2 for line in wrap ]
```

Comment:
Please remove this line, if it is not used.

Reply:
@afuetterer Done.